drivers/gles3/storage/particles_storage.cpp

```diff
@@ -357,12 +357,12 @@ void ParticlesStorage::particles_restart(RID p_particles) {
 
 void ParticlesStorage::particles_set_subemitter(RID p_particles, RID p_subemitter_particles) {
 	if (p_subemitter_particles.is_valid()) {
-		WARN_PRINT_ONCE_ED("The GL Compatibility rendering backend does not support particle sub-emitters.");
+		WARN_PRINT_ONCE_ED("The Compatibility renderer does not support particle sub-emitters.");
```

Comment:
The hyphen inconsistency also applies to "sub-emitter", but for some reason the class reference heavily favours "subemitter". So this is not a suggestion, just a note.

Reply:
The manual has a heavy preference for "sub-emitter". First one of these hyphen inconsistencies I've found where there is a large mismatch between the two.

Reply:
More context: The editor itself, which should be the source of truth, calls it a "Sub Emitter" or "sub_emitter" for the name in the inspector on a GPUParticles3D. And the rendering server exposes the relevant methods with "subemitter" in the names, rather than "sub_emitter".

Reply:
With no obvious standard here, I think deferring this change for later makes sense.

Reply:
On the other hand you have "sub-resource" that suffers a similar issue. Worth tackling separately.